bin/app.ts

```diff
@@ -93,8 +97,8 @@ export class RoutingAPIPipeline extends Stack {
     // Secrets are stored in secrets manager in the pipeline account. Accounts we deploy to
     // have been granted permissions to access secrets via resource policies.
 
-    const infuraProjectId = sm.Secret.fromSecretAttributes(this, 'InfuraProjectId', {
-      secretCompleteArn: 'arn:aws:secretsmanager:us-east-2:644039819003:secret:infuraProjectId-UlSwK2',
+    const jsonRpcProvider = sm.Secret.fromSecretAttributes(this, 'jsonRpcProvider', {
+      secretCompleteArn: 'arn:aws:secretsmanager:us-east-2:644039819003:secret:jsonRpcProvider-UlSwK2',
```

Comment:
Did you create the secret in secrets manager? I don't see it there

Reply:
https://github.com/Uniswap/routing-api/pull/47/files

Hey reworked and simplified this PR entirely